Update supported-azure-ad-features.md (#87478)
Azure/azure-content: articles/active-directory-b2c/supported-azure-ad-features.md

|Feature  |Azure AD  | Azure AD B2C |
|---------|---------|---------|

> | [Groups](../active-directory/fundamentals/active-directory-groups-create-azure-portal.md) | Groups can be used to manage administrative and user accounts.| Groups can be used to manage administrative accounts. [Consumer accounts](user-overview.md#consumer-user) can not be member of any group. Groups cannot be used for [group-based assignment of enterprise applications](https://docs.microsoft.com/en-us/azure/active-directory/manage-apps/assign-user-or-group-access-portal)|

Reviewer: ```suggestion
| [Groups](../active-directory/fundamentals/active-directory-groups-create-azure-portal.md) | Groups can be used to manage administrative and user accounts.| Groups can be used to manage administrative accounts. [Consumer accounts](user-overview.md#consumer-user) can't be member of any group, and so you can't perform [group-based assignment of enterprise applications](../active-directory/manage-apps/assign-user-or-group-access-portal.md)|
```

Reviewer: @jasonfritts Please see my suggested commit. Thanks.
